From 036ccc36399a124eaa800c7acfdb6cea88e23a74 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Mon, 30 Nov 2020 01:21:33 +0200
Subject: [PATCH] [SW-0] gaudi: (re)use pcie wrap header

Do not redefine macros which can be included from specs.
Remove include of PCIe WRAP DRAIN CFG registers.

Change-Id: Ifbc0fe87f5ef331c4f543e2fe58dca4450de782e
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/gaudi/asic.c b/gaudi/asic.c
index d2af790..3f31b4f 100644
--- a/gaudi/asic.c
+++ b/gaudi/asic.c
@@ -6,6 +6,7 @@
  */
 
 #include <psoc_global_conf_regs.h>
+#include <pcie_wrap_regs.h>
 
 #include <string.h>
 #include <preboot.h>
diff --git a/gaudi/common.h b/gaudi/common.h
index 30f2288..2da4e04 100644
--- a/gaudi/common.h
+++ b/gaudi/common.h
@@ -20,8 +20,6 @@
 #define PPBOOT_TEXT_BASE		0x7FF0000000
 #define PREBOOT_TEXT_BASE		(PPBOOT_TEXT_BASE + PPBOOT_SIZE)
 
-#define mmPCIE_WRAP_HBW_DRAIN_CFG	0xC01D54
-#define mmPCIE_WRAP_LBW_DRAIN_CFG	0xC01D5C
 #define mmPSOC_CPU_PLL_BASE		0xC70000
 #define mmPSOC_PCI_PLL_BASE		0xC72000
 #define mmPSOC_TPC_PLL_BASE		0xC73000
